### PR TITLE
added: rules, questions and score page

### PR DESCRIPTION
**Pull request by rkapoor10**
- Rules Page
rules

- Questions Page
question

- Score Page
scorecard

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: questions/questions.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<body>
+    <div class="flex-column-center">
+        <a class="flex-column-center margin-bottom-4" href="/index.html">
+            <img class="hero-img-small" src="/assets/2904297-ff9800.svg" alt="hero-img">
+            <h1>kwizz<span class="darkyellow">it!</span></h1>
+        </a>
+        <div class="question-box flex-column-center center">
+            <h2>QUIZ</h2>
+            <div class="quiz-status space-between m-2 txt-m ">
+                <p>Question:<span class="darkyellow"> 1/5</span></p>
```
Comment (Priya-Patel31) [inline questions/questions.html:26]: don't use non-semantic tag

File: scorecard/scorecard.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                </ul>
+            </div>
+        </div>
+        <a class="btn-grad position-main-btn" href="/categories/category.html">PLAY NEXT!</a>
+    </div>
+    <footer class="site-footer">
+        <div class="fs-1r fw-semibold">Made with <span class="baseblue"><a
+                    href="https://swift-uiv1.netlify.app/">SWIFT-UI </a> </span> by RK </div>
+        <ul class="flex-icons">
+            <li class="fs-15 footer-icons"><a class="" href="https://github.com/rkapoor10/quiz-app"><i
```
Comment (Priya-Patel31) [inline scorecard/scorecard.html:85]: remove unused class

File: rules/rules.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <h2 class="darkyellow">#RULES</h2>
+            <p class="txt-s gray">Before starting the quiz read the rules 😎</p>
+            <hr>
+            <ul class="txt-s white">
+                <li class="m-1"><span class="darkyellow m-1">>></span> This is a quiz on Blacklist so do watch it or you can
+                    try you luck!</li>
+                <li class="m-1"><span class="darkyellow m-1">>></span> Quiz contains 5 questions with a single correct
+                    answer</li>
+                <li class="m-1"><span class="darkyellow m-1">>></span> For every single <span class=" darkyellow">correct
+                        answer</span> you get<span class="darkyellow"> +4 </span> </li>
```
Comment (Priya-Patel31) [inline rules/rules.html:33]: avoid using non-semantic tag

**Review by Priya-Patel31 — APPROVED** [review]
Everything looks good and code is quite clean . Go ahead and merge it !!

File: questions/questions.html (hunk 1 above)
Comment (rkapoor10) [reply questions/questions.html:26]: will keep in mind

File: scorecard/scorecard.html (hunk 2 above)
Comment (rkapoor10) [reply scorecard/scorecard.html:85]: done

File: rules/rules.html (hunk 3 above)
Comment (rkapoor10) [reply rules/rules.html:33]: ok will keep in mind

**Pull request merged**